tests/framework/extensions/provisioning/creates.go

```diff
@@ -227,8 +227,8 @@ func CreateProvisioningCustomCluster(client *rancher.Client, externalNodeProvide
 		}
 	}
 
-	if clustersConfig.Hardened && clustersConfig.KubernetesVersion <= string(provisioninginput.HardenedKubeVersion) {
-		err = hardening.HardeningNodes(client, clustersConfig.Hardened, nodes, rolesPerNode)
+	if clustersConfig.Hardened {
```

Comment:
I would just add an 'if config contains 'k3s' ... ' instead of a whole new function

Reply:
Resolved.